.gitignore

```diff
@@ -7,7 +7,7 @@ config/*
 !config/default.yaml
 
 # Draupnir test data
-mjolnir-registration.yaml
+draupnir-registration.yaml
```

Comment:
(just picking a random line of this change)

Make sure this is in the release notes please as it is potentially a breaking change and would break bots if not updated downstream

Reply:
Would it break bots? As far as we know, the path is never referenced directly or implicitly, you have to explicitly provide the path to the registration file to draupnir in the cli, and this is just to generate -- hmm.

@FSG-Cat does the documentation make reference to `mjolnir-registration.yaml` and ask you to move / copy it?

Reply:
Docs never refers to any file called Draupnir or mjolnir that is pre existing. The file that we talk about in docs is created by Draupnir on demand with a file name of our choosing or the docs are wrong currently. (Remember D4A mdad and K8s both use synthetic Reg files that are completely made up by the maintainers of said deployment methods and not generated by Draupnir)

Reply:
This file as far as i can tell is a file that mxtester needs but well i accounted for that.

The more likely to break file would be our entrypoint file if someone is refering to that for some reason. But thats already accounted for as a change that we need to tell people about.

Reply:
@FSG-Cat Could you add a `CHANGELOG.md` from the template https://github.com/Gnuxie/tsconfig/blob/main/templates/CHANGELOG.md

Add details for both the entrypoint change and the generated registration file change.

Then at the bottom:

```
## Versions v2.0.0-beta.7 and prior

Please see  https://github.com/the-draupnir-project/Draupnir/releases
```

I don't know if we will use this but it's going to at least help us keep unreleased changes in check.

Reply:
I think its a great idea.

Reply:
Turns out I _thought_ it was in https://github.com/the-draupnir-project/Draupnir/blob/MTRNord/draupnir4all/docs/appservice.md but a) it wasnt and b) I already used draupnir there...